packages/cdk/Makefile

```diff
@@ -15,4 +15,6 @@ init:
 	npm ci --silent
 
 release:
+	cd ./lib/wes_adapter/ && ./install.sh
+	cd ../..
```

Comment:
I think there is a better (more reliable way) of doing this:
```
( cd ./lib/wes_adapter/ && ./install.sh )
```

Reply:
Done, thanks!